pkg/csi/service/wcpguest/controller.go

```diff
@@ -452,6 +467,91 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 	timeoutSeconds := int64(getAttacherTimeoutInMin(ctx) * 60)
 	timeout := time.Now().Add(time.Duration(timeoutSeconds) * time.Second)
 
+	var isFileVolume bool
```

Comment:
Var definition can be moved to line 478.

Reply:
Ok